packages/components/Tabs/src/Tabs.tsx

```diff
@@ -66,13 +69,18 @@ export const Tabs = compose<TabsType>({
     // Stores views to be displayed
     const map = new Map<string, React.ReactNode[]>();
 
+    const pressable = useAsPressable({
+      ...rest,
+    });
+
```

Comment:
Why is this only used by windows? as far as I can tell? Would this not be used by the other platforms too?

Reply:
It is only used by windows as it was needed to implement the keyboarding behavior. For the spec'd behavior, I believe that the other platforms do not need it.  I will be moving it under the windows platform check to make it clear it is only for windows.